src/vs/code/electron-sandbox/issue/issueReporterMain.ts

```diff
@@ -715,25 +731,36 @@ export class IssueReporter extends Disposable {
 		hide(problemSource);
 		hide(extensionSelector);
 
-		if (issueType === IssueType.Bug) {
-			show(problemSource);
+		show(problemSource);
+		show(titleTextArea);
+		show(descriptionTextArea);
 
+		if (fileOnExtension) {
+			show(extensionSelector);
+		}
+
+		if (fileOnExtension && selectedExtension?.hasIssueUriRequestHandler) {
+			hide(titleTextArea);
+			hide(descriptionTextArea);
+			reset(descriptionTitle, localize('handlesIssuesElsewhere', "This extension handles issues outside of VS Code"));
+			reset(descriptionSubtitle, localize('elsewhereDescription', "The '{0}' extension prefers to use an external issue reporter. To be taken to that issue reporting experience, click the button below.", selectedExtension.displayName));
+			this.previewButton.label = localize('openIssueReporter', "Open Issue Reporter");
```

Comment:
I feel like we should have this say `External` or something that makes it clear this isn't the issue reporter that you're currently in for those who don't read the description